web/src/app/admin/connectors/document360/page.tsx

```diff
@@ -0,0 +1,244 @@
+import { adminDeleteCredential, linkCredential } from "@/lib/credential";
+import { CredentialForm } from "@/components/admin/connectors/CredentialForm";
+import {
+  TextFormField,
+  TextArrayFieldBuilder,
+} from "@/components/admin/connectors/Field";
+import { ConnectorsTable } from "@/components/admin/connectors/table/ConnectorsTable";
+import { ConnectorForm } from "@/components/admin/connectors/ConnectorForm";
+import { usePublicCredentials } from "@/lib/hooks";
+
+const MainSection = () => {
+	const { mutate } = useSWRConfig();
+	const {
+		data: connectorIndexingStatuses,
+		isLoading: isConnectorIndexingStatusesLoading,
+		error: isConnectorIndexingStatusesError,
+	} = useSWR<ConnectorIndexingStatus<any, any>[]>(
+		"/api/manage/admin/connector/indexing-status",
+		fetcher
+	);
+
+	const {
+		data: credentialsData,
+		isLoading: isCredentialsLoading,
+		error: isCredentialsError,
+		refreshCredentials,
+	} = usePublicCredentials();
+
+	if (
+		(!connectorIndexingStatuses && isConnectorIndexingStatusesLoading) ||
+		(!credentialsData && isCredentialsLoading)
+	) {
+		return <LoadingAnimation text="Loading" />;
+	}
+
+	if (isConnectorIndexingStatusesError || !connectorIndexingStatuses) {
+		return <div>Failed to load connectors</div>;
+	}
+
+	if (isCredentialsError || !credentialsData) {
+		return <div>Failed to load credentials</div>;
+	}
+
+	const document360ConnectorIndexingStatuses: ConnectorIndexingStatus<
+		Document360Config,
+		Document360CredentialJson
+	>[] = connectorIndexingStatuses.filter(
+		(connectorIndexingStatus) =>
+			connectorIndexingStatus.connector.source === "document360"
+	);
+
+  const document360Credential: Credential<Document360CredentialJson> | undefined = 
+    credentialsData.find((credential) => credential.credential_json?.document360_api_token);
+
+  return (
+    <>
+      <h2 className="font-bold mb-2 mt-6 ml-auto mr-auto">
+        Step 1: Provide Credentials
+      </h2>
+      {document360Credential ? (
+        <>
+          <div className="flex mb-1 text-sm">
+            <p className="my-auto">Existing Document360 API Token: </p>
+            <p className="ml-1 italic my-auto">
+              {document360Credential.credential_json.document360_api_token}
+            </p>
+            <button
+              className="ml-1 hover:bg-gray-700 rounded-full p-1"
+              onClick={async () => {
+                await adminDeleteCredential(document360Credential.id);
+                refreshCredentials();
+              }}
+            >
+              <TrashIcon />
+            </button>
+          </div>
+        </>
+      ) : (
+        <>
+          <p className="text-sm mb-4">
+            To use the Document360 connector, you must first provide the API token 
+            and portal ID corresponding to your Document360 setup. For more details, 
+            see the <a className="text-blue-500" href="https://docs.document360.io/">official Document360 documentation</a>.
```

Comment:
https://apidocs.document360.com/apidocs/api-token

Can we link to this instead?

Reply:
Okay, I will. great catch.